docs/.vuepress/config.js

```diff
@@ -67,6 +67,15 @@ module.exports = {
           "/sdk/sdk-by-examples/simple-governance/running-the-application"
         ]
       },
+      {
+        title: "Light Client",
+        collapsable: false,
+        children: [
+	  "/light/",
+	  "/light/getting_started",
+	  "/light/load_balancer"
```

Comment:
We should delete load balancer from here since it’s a spec:
```suggestion
```

Reply:
fixed